core/drivers/scmi-msg/clock.c

```diff
@@ -303,6 +303,12 @@ static void scmi_clock_describe_rates(struct scmi_msg *msg)
 		status =  plat_scmi_clock_rates_array(msg->agent_id, clock_id,
 						      plat_rates, &ret_nb);
 		if (status == SCMI_SUCCESS) {
+			assert((sizeof(p2a) % sizeof(uint32_t)) == 0U);
```

Comment:
`COMPILE_TIME_ASSERT()`?

Reply:
expected true by design of the SCMI spec.
Failure on alignment is expected to catch unaligned accesses.
If `CFG_SCTLR_ALIGNMENT_CHECK=y`.